### PR TITLE
feature/CWALL-257_cloud-wallet-url-support

### PR DESCRIPTION
**Pull request by sanderPostma**
For screen shots see https://sphereon.atlassian.net/browse/CWALL-257

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,10 +39,10 @@
   },
   "devDependencies": {
     "concurrently": "^1.0.0",
-    "husky": "^9.0.11",
+    "husky": "^9.1.6",
     "prettier": "^3.3.3",
-    "rimraf": "^5.0.9",
-    "typescript": "4.9.5"
+    "rimraf": "^5.0.10",
+    "typescript": "^5.6.3"
```
Comment (BtencateSphereon) [inline package.json]: we go from a fixed version to a flexible one?

File: package.json [hunk 2]
```diff
@@ -52,25 +52,25 @@
...
+      "@sphereon/ssi-sdk.oid4vci-issuer": "0.30.2-next.221",
+      "@sphereon/ssi-sdk.oid4vci-issuer-rest-api": "0.30.2-next.221",
+      "@sphereon/ssi-express-support": "0.30.2-next.221",
+      "@sphereon/ssi-sdk.issuance-branding": "0.30.2-next.221",
+      "@sphereon/ssi-sdk.siopv2-oid4vp-op-auth": "0.30.2-next.221",
+      "@sphereon/ssi-sdk.siopv2-oid4vp-rp-auth": "0.30.2-next.221",
+      "@sphereon/ssi-sdk.ebsi-support": "0.30.2-next.221",
+      "@sphereon/ssi-sdk-ext.key-utils": "0.24.1-unstable.137",
+      "@sphereon/ssi-sdk-ext.did-utils": "0.24.1-unstable.137",
+      "@sphereon/oid4vci-issuer": "0.16.1-feature.sdk.38.rfc8414.auth.server.metadata.172",
```
Comment (BtencateSphereon) [inline package.json]: do we have a next version?

File: packages/agent/package.json [hunk 3]
```diff
@@ -143,23 +144,23 @@
...
-    "eslint": "^9.7.0",
+    "eslint": "^9.13.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-import": "^2.31.0",
     "ts-node": "^10.9.2",
-    "tsx": "^4.16.5",
+    "tsx": "^4.19.1",
     "typescript": "4.9.5"
```
Comment (BtencateSphereon) [inline packages/agent/package.json]: this is still using 4 where the main is using 5

File: packages/agent/package.json [hunk 4]
```diff
@@ -42,59 +42,60 @@
   "dependencies": {
     "@decentralized-identity/ion-sdk": "^0.6.0",
     "@ethersproject/abstract-signer": "^5.7.0",
+    "@sphereon/did-auth-siop": "0.16.1-feature.sdk.38.rfc8414.auth.server.metadata.172",
```
Comment (BtencateSphereon) [inline packages/agent/package.json]: use a next version?

File: packages/web-wallet/package.json [hunk 5]
```diff
@@ -20,93 +20,94 @@
...
     "@refinedev/core": "4.51.0",
-    "@refinedev/devtools": "^1.2.6",
+    "@refinedev/devtools": "^1.2.9",
     "@refinedev/react-hook-form": "4.8.20",
     "@refinedev/react-router-v6": "4.5.11",
     "@refinedev/simple-rest": "5.0.8",
     "@refinedev/supabase": "5.9.0",
-    "@sphereon/did-auth-siop": "0.16.1-feature.jarm.sdk.137",
-    "@sphereon/did-auth-siop-adapter": "0.16.1-feature.jarm.sdk.137",
+    "@sphereon/did-auth-siop": "0.16.1-feature.sdk.38.rfc8414.auth.server.metadata.172",
```
Comment (BtencateSphereon) [inline packages/web-wallet/package.json]: use a next?

File: packages/web-wallet/package.json [hunk 6]
```diff
@@ -117,31 +118,31 @@
...
+    "husky": "^9.1.6",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "prettier": "^3.3.3",
     "sass-loader": "^14.2.1",
     "style-loader": "^3.3.4",
-    "ts-jest": "^29.2.2",
+    "ts-jest": "^29.2.5",
     "ts-node": "^10.9.2",
     "typescript": "4.9.5",
```
Comment (BtencateSphereon) [inline packages/web-wallet/package.json]: still using 4, main is using 5

File: packages/web-wallet/public/locales/en/common.json [hunk 7]
```diff
@@ -395,7 +400,7 @@
   "credential_catalog_column_credential_description_label": "Description",
   "credential_catalog_column_actions_label": "Actions",
   "credential_catalog_card_view_display_mode": "Card view",
-  "credential_catalog_list_view_display_mode": "List view"
+  "credential_catalog_list_view_display_mode": "List view",
```
Comment (BtencateSphereon) [inline packages/web-wallet/public/locales/en/common.json:403]: this one is missing in the dutch localization file

**Review by BtencateSphereon — CHANGES_REQUESTED** [review]

File: packages/agent/package.json (hunk 3 above)
Comment (sanderPostma) [reply packages/agent/package.json]: fixed

File: packages/web-wallet/package.json (hunk 6 above)
Comment (sanderPostma) [reply packages/web-wallet/package.json:147]: I do not understand the statement. we do not use the main branch

File: packages/web-wallet/public/locales/en/common.json (hunk 7 above)
Comment (sanderPostma) [reply packages/web-wallet/public/locales/en/common.json:403]: fixed, was not the only one there

File: packages/agent/package.json (hunk 4 above)
Comment (sanderPostma) [reply packages/agent/package.json]: By now yes. Fixed

File: packages/web-wallet/package.json (hunk 6 above)
Comment (sanderPostma) [reply packages/web-wallet/package.json]: fixed

File: package.json (hunk 1 above)
Comment (sanderPostma) [reply package.json]: fixed

File: package.json (hunk 2 above)
Comment (sanderPostma) [reply package.json]: by now we have

File: packages/web-wallet/package.json (hunk 5 above)
Comment (sanderPostma) [reply packages/web-wallet/package.json]: fixed

**Review by BtencateSphereon — APPROVED** [review]

**Pull request merged**